packages/calcite-components/src/components/flow-item/flow-item.tsx

```diff
@@ -132,11 +132,11 @@ export class FlowItem extends LitElement implements InteractiveComponent {
   /** Specifies the size of the component. */
   @property({ reflect: true }) scale: Scale = "m";
 
-  /** When true, flow-item is displayed within a parent flow. */
+  /** When present, flow-item is displayed within a parent flow. */
```

Comment:
another potential follow-up: Should we use "component" here instead of "flow-item" for consistency?